syft/messaging/message.py

```diff
@@ -218,9 +211,14 @@ def bufferize(worker: AbstractWorker, action_message: "CommandMessage") -> "Comm
             data = bufferize(message)
         """
         protobuf_op_msg = CommandMessagePB()
-        protobuf_op = ComputationAction.bufferize(worker, action_message.action)
 
-        protobuf_op_msg.action.CopyFrom(protobuf_op)
+        if isinstance(action_message.action, ComputationAction):
+            protobuf_op = ComputationAction.bufferize(worker, action_message.action)
+            protobuf_op_msg.computation.CopyFrom(protobuf_op)
+        elif isinstance(action_message.action, CommunicationAction):
+            protobuf_op = CommunicationAction.bufferize(worker, action_message.action)
+            protobuf_op_msg.communication.CopyFrom(protobuf_op)
```

Comment:
As long as `ComputationAction` and `CommunicationAction` are included in the serde's list of classes with their own serialization (and it looks like they are), I think you can call `sy.serde.protobuf.serde._serialize(action_message.action)` here without having to check types.